src/Arcus.Testing.Tests.Unit/Assert_/AssertCsvTests.cs

```diff
@@ -240,6 +240,61 @@ public void CompareWithIgnoredRowAndColumnOrder_WithShuffledCsv_Succeeds()
             });
         }
 
+        [Property]
+        public void CompareWithIgnoredColumnOrderAndColumnIndexes_WithShuffledCsv_FailsWithDescription()
+        {
+            // Arrange
+            TestCsv expected = TestCsv.Generate();
+            TestCsv actual = expected.Copy();
+            int[] possibleIndexes = Enumerable.Range(0, expected.ColumnCount).ToArray();
+            int ignoredIndex = Bogus.PickRandom(possibleIndexes);
+
+            actual.ShuffleColumns();
+
+            CompareShouldFailWithDescription(
+                expected,
+                actual,
+                options =>
+                {
+                    options.ColumnOrder = AssertCsvOrder.Ignore;
+                    options.IgnoreColumnIndex(ignoredIndex);
+                },
+                "cannot compare",
+                "column indexes can only be ignored when column order is included",
```

Comment:
The reason why we have a `param string[] errorParts`, is bc we do not want the test to fail if the error message was changed slightly but is still talking about the same failure. In this case, there is an entire error message here, maybe if use `"cannot compare", "column indexes", "column order", "included"` is enough. If you understand what I mean? That way, you can change the structure of the error message, without the test to fail.

Reply:
Fixed in [cc29d93](https://github.com/arcus-azure/arcus.testing/pull/161/commits/cc29d933465e61b047c0a1bf4c9453994396ff41)